tests/conftest.py

```diff
@@ -109,6 +109,17 @@ def conn_pool_to_second_graph_service(pytestconfig):
     yield pool
     pool.close()
 
+
+@pytest.fixture(scope="session")
+def conn_pool_to_third_graph_service(pytestconfig):
```

Comment:
Is the third graphd service connection pool necessary to test your cases? Why not to use the existing conn pool?

Reply:
the existing conn pool is root account，my cases need guest account